samples/dfu/ab_split/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(ab_split)
+
+target_sources(
```

Comment:
the `app PRIVATE` should be on the line with `target_sources`, it is the file/folder names that go onto different lines, same with below and with the ifdef one also